Update config.xml.in

Remove now unneeded 4A binding dependency, and add new required audio
permission.

Upstream-Status: Pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/platforms/agl/alexa-voiceagent-service/conf.d/wgt/config.xml.in b/platforms/agl/alexa-voiceagent-service/conf.d/wgt/config.xml.in
index 3486518..14fd6a2 100644
--- a/conf.d/wgt/config.xml.in
+++ b/conf.d/wgt/config.xml.in
@@ -9,6 +9,7 @@
 
     <feature name="urn:AGL:widget:required-permission">
         <param name="urn:AGL:permission::public:no-htdocs" value="required" />
+        <param name="urn:AGL:permission::public:audio" value="required" />
         <param name="urn:AGL:permission:audio:public:audiostream" value="required" />
         <param name="urn:AGL:permission::public:hidden" value="required" />
         <param name="urn:AGL:permission::system:run-by-default" value="required" />
@@ -23,7 +24,6 @@
     </feature>
 
     <feature name="urn:AGL:widget:required-api">
-        <param name="ahl-4a" value="ws" />
         <param name="vshl-capabilities" value="ws" />
 	  </feature>
-</widget>
\ No newline at end of file
+</widget>
